pom.xml

```diff
@@ -117,7 +110,6 @@
       <plugins>
         <plugin>
           <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
@recena I was the one suggesting to leave it as it is, to not change the current behaviour. Any particular reason to not allow maven2?